### ISSUE TITLE
firebase appdistribution:distribute attempts to run tests even though --test-devices flag was not used

### ISSUE THREAD

**Issue by sofi-kry** [firebase/firebase-tools#6749]
### [REQUIRED] Environment info

**firebase-tools:** 13.2.0

**Platform:** Ubuntu

### [REQUIRED] Test case

```
firebase appdistribution:distribute x.apk --app id  --groups "android-internal-testers" --release-notes "notes"
```

### [REQUIRED] Steps to reproduce

```
firebase appdistribution:distribute x.apk --app id  --groups "android-internal-testers" --release-notes "notes"
```

### [REQUIRED] Expected behavior

Automated tests are not run, given that the flag `--test-devices` is not included. The job ends successfully.

### [REQUIRED] Actual behavior

<img width="447" alt="Screenshot 2024-02-07 at 14 27 52" src="https://github.com/firebase/firebase-tools/assets/93991979/49764958-a63c-403f-a226-0aafc47660a3">

**Comment by MaesterChestnut** [comment]
Apologies for this issue. Releases are currently being uploaded despite the error message. We're currently working on a fix, but in the meantime please use 13.1.0 to remove the error message. We'll post up to date information on our status page: https://status.firebase.google.com/

**Comment by MaesterChestnut** [comment]
We just pushed a fix so that firebase-tools@13.2.0 should no longer throw unexpected errors while uploading releases. Please try again to confirm @sofi-kry

**Comment by sofi-kry** [comment]
It works now, thank you!
